modules/virt-about-reviewing-resource-consumption.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-reviewing-resource-consumption-by-pods.adoc
+//
+
+[id="virt-about-reviewing-resource-consumption-by-pods_{context}"]
+
+= About reviewing resource consumption by pods
+
+Line chart graphs in the {VirtProductName} dashboard present data for virtual machines that are consuming the most resources within each pod for a selected time range.
```

Comment:
@lmandavi Maybe:
Line chart graphs in the {VirtProductName} dashboard present data for virtual machines or `virt-launcher` pods that are consuming the most resources for a selected time range.

Reply:
We simplified the heading:
About reviewing resource consumption.